pylana/v2.py

```diff
@@ -6,11 +6,13 @@
 """
 
 from pylana.logs import LogsAPI
+from pylana.users import UsersAPI
 from pylana.resources import ResourceAPI
 from pylana.shiny_dashboards import ShinyDashboardAPI
+from pylana.dashboards import DashboardAPI
```

Comment:
It's nice to order imports alphabetically. PyCharm does this with control + option + o (for "optimise imports"), when the caret is in the imports. It also deletes unused imports this way.